.storybook/privatePresets/ignoreTestFilesPreset.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Ignore tests preset
+ *
+ * Modify webpack configuration to exclude all files with `.spec` or `.test` in
+ * them. We use webpack's dynamic require to generate stories for Icons; which
+ * inadvertantly causes `Icons.spec.tx` to be included. This causes storybook to
```

Comment:
Looks like we filter out `spec` files here (should we also filter out .test files?), not quite sure how this preset ends up being needed but #notawebpackguru.  https://github.com/apollographql/space-kit/pull/150/commits/97b2cea6b42d8983099c9a7c080f376389843ea6#diff-a233c537848b6cd0d08d7c4a96487ae7R18

Reply:
Great question @timbotnik . This PR adds a `src/Icons/Icons.spec.tsx` and one for illustrations as well. Those use `fs` to read all the icons files and runs tests against them all. The storybook story, `Icons.story.tsx` (compiled with webpack), uses `require(` ... `)`, which will look at all possible matches and transpile them so the dynamic includes will work. This fails when trying to dynamically transpile the files using `fs`.

Reply:
Also, yes. We should also filter out .test files. Thanks 👍